app/code/Transoft/Blog/Model/DataProvider.php

```diff
@@ -0,0 +1,73 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Model;
+
+use Magento\Framework\Filesystem\Io\File;
+use Magento\Ui\DataProvider\AbstractDataProvider;
+use Transoft\Blog\Model\ResourceModel\Post\CollectionFactory;
+
+/**
+ * Class DataProvider
+ *
+ * Generates data provider
+ */
+class DataProvider extends AbstractDataProvider
+{
+    protected $collection;
+    protected $_loadedData;
```

Comment:
**private**

Reply:
fixed

Reply:
really?

Reply:
sorry, missed that, i thought '_loadedData' should be still protected.